.github/dependabot.yml

```diff
@@ -12,3 +12,58 @@ updates:
       patterns:
         - "*"
 
+
+- package-ecosystem: docker
+  directory: /
+  schedule:
+    interval: daily
+
+- package-ecosystem: docker
+  directory: /tools/c7n_mailer
+  schedule:
+    interval: daily
+
+- package-ecosystem: pip
+  directory: /tools/c7n_oci
+  schedule:
+    interval: daily
+
+- package-ecosystem: docker
+  directory: /tools/c7n_org
+  schedule:
+    interval: daily
+
+- package-ecosystem: docker
+  directory: /tools/c7n_policystream
+  schedule:
+    interval: daily
+
+- package-ecosystem: pip
+  directory: /tools/c7n_salactus
+  schedule:
+    interval: daily
+
+- package-ecosystem: gomod
+  directory: /tools/cask
+  schedule:
+    interval: daily
```

Comment:
the frequency on these is way too high, and is extraneous to our existing workflow for updating dependencies.

Reply:
Do you mean specifically daily is to frequent for /tools/casks or daily is too frequent for any of the dependencies ?

Reply:
for any of them, for most of the python dependencies, we have makefile targets that we use prior to release, to pull in the latest releases (`make pkg-rebase`).. in general for automated dep prs, we want to target monthly. we handle dependabot security updates out of band, buts its mostly immaterial till we have a release, as previous releases are on a frozen dependency graph for repeatable installation.

Reply:
i do think we want the monthly cadence for the golang tools

Reply:
We can change the frequency to monthly as well